tests/unit/utils/utils.ts

```diff
@@ -265,6 +266,27 @@ export function createVoucherDrainedEvent(voucher: Address, amount: BigInt): Vou
     return event;
 }
 
+export function createVoucherRefundedEvent(voucher: Address, amount: BigInt): VoucherRefunded {
+    let mockEvent = newMockEvent();
+    let event = new VoucherRefunded(
+        mockEvent.address,
+        mockEvent.logIndex,
+        mockEvent.transactionLogIndex,
+        mockEvent.logType,
+        mockEvent.block,
+        mockEvent.transaction,
+        mockEvent.parameters,
+        mockEvent.receipt,
+    );
+
+    event.parameters = EventParamBuilder.init()
+        .address('voucher', voucher)
```

Comment:
```suggestion
        .address('voucher', Address.fromString(voucher))
```
To make calling the function easier.
BTW this can be applied to all create*Event() functions.